librocksdb_sys/src/lib.rs

```diff
@@ -82,6 +82,8 @@ pub enum DBCompressionType {
     DBLz4hc = 5,
     // DBXpress = 6, not support currently.
     DBZstd = 7,
+
+    DBZstdNotFinal = 0x40,
```

Comment:
This enum variant is useless, better remove. Since unmatched enum variant panics, ok to skip range check.